remi/gui.py

```diff
@@ -2733,8 +2733,9 @@ def __init__(self, text, **kwargs):
             "sendCallback('%s','%s');" \
             "event.stopPropagation();event.preventDefault();" % (self.identifier, self.EVENT_ONCLICK)
         self.set_text(text)
-        self.treeopen = False
-        self.attributes['treeopen'] = 'false'
+        treeopen = kwargs.get('treeopen', False)
```

Comment:
It could be ok to have a **treeopen** parameter, but it's better to have it as an explicit optional parameter.